### DISCUSSION TITLE
Info on Xamarin Android commercial version

### DISCUSSION THREAD

**Discussion by MattiaDurli** [emgucv/emgucv#468] [General]
Hello,

I just bought the Xamarin Android commercial version.

Link download:
libemgucv-android-4.5.1.4349.zip
Link valid until:
25-apr-2021 14:51:43 CET

I have two questions:

1) the link will expire, how can I download the minor updates that will be published during my year licence?

2) I used successfully EMGU by referencing the .dll, but how can I use it as Nuget repository? The information in the wiki says:
_The commercial download page also provide access to the "Emgu.CV.runtime.android" nuget pacakge. Similar to the "Emgu.CV.runtime.windows" nuget package that provide runtime dependencies for Windows, it provides the runtime dependencies for Emgu CV targeting Andorid application. Once you download the nuget package, you can either put it in your own nuget server, or you can put it in a local folder and configure Visual Studio to use the folder as a local repository._
Where can I find Emgu.CV.runtime.windows?

Thanks
PS I ask here because I wrote to support@emgu.com 10 days ago but have not received any answer

**Comment by emgucv** [comment]
Hi Mattia, 

We have replyed to your email on March 29, 10:41am EST, the email is sent from canming@emgu.com If it is blocked by your company's email server please add that email address to the "allow list" for future communication. Thanks.

Below is part of the response in the email we sent:

> Our e-commercial provider only host the file for a short period of time. To provide you access to future updates, we have hosted the files in our own user download session. Detail instructions to find the Url and credentials from your email receipt can be found here:
> https://www.emgu.com/wiki/index.php/Download_And_Installation#Commercial_Release
> 
> You will find the “Emgu.CV.runtime.android” nuget package once you logged into our website.

**Reply by MattiaDurli** [reply]
Thanks! found your mail in the gmail spam folder
